rafthttp/stream.go

```diff
@@ -392,45 +392,52 @@ func (cr *streamReader) dial(t streamType) (io.ReadCloser, error) {
 	lv := semver.Must(semver.NewVersion(version.Version))
 	if compareMajorMinorVersion(rv, lv) == -1 && !checkStreamSupport(rv, t) {
 		resp.Body.Close()
+		cr.picker.unreachable(u)
```

Comment:
Based on our previous discussion, as long as it doesn't function as expectation, it is unreachable in the view of local etcd. So I report `unreachable` to the picker.